bin/zbrewmount

```diff
@@ -0,0 +1,185 @@
+#!/bin/sh
+
+#
+# Temporary hack - replace with a manpage
+#
+syntax() {
+	echo "\
+Syntax:
+	zbrewmount {-m|-u} {-s|-t} [-v] sw
+Definition:
+    -m:  to mount
+    -u:  to unmount
+    -s:  action on source
+    -t:  action on target
+    -v:  enable verbose
+    sw:  product ID
+    
+Note:
+    must specify either -m or -u, not both
+    must specify either -s or -t, not both
+    -v is optional
+    sw is mandatory
+    
+Examples:
+    Mount IGY630 at ZBREW_SRC_ZFSROOT
+        zbrewmount -m -s IGY630
+    Unmount EQAE20 at ZBREW_TGT_ZFSROOT as well as ZBREW_TGT_HLQ
+        zbrewmount -u -t EQAE20
+" >&2
+    exit 8
+}
+
+# ---------------------------------------------------------------------
+# some options cannot be appeared in multiple times or combining with others
+# ---------------------------------------------------------------------
+function opt_combine_repeat_chk
```

Comment:
is this function still called? If not, do you need to add a check for src/tgt both specified and mount/unmount both specified in the mainline?